src/elements/form/Control.ts

```diff
@@ -39,52 +39,52 @@ import { Helpers } from './../../utils/Helpers';
                         <div class="novo-control-input {{control.controlType}}" [ngSwitch]="control.controlType" [attr.data-automation-id]="control.key">
                             <!--Text-based Inputs-->
                             <!--TODO prefix/suffix on the control-->
-                            <div class="novo-control-input-container novo-control-input-with-label" *ngSwitchCase="'textbox'">
+                            <div class="novo-control-input-container novo-control-input-with-label" *ngSwitchCase="'textbox'" [tooltip]="control.tooltip" [tooltipPosition]="control.tooltipPosition">
```

Comment:
It will still go into the BaseControl and that, since we create those controls and make the NovoFormControl via them.